### ISSUE TITLE
UC1/US6 Look for ways to make better use of existing data on soils

### ISSUE THREAD

**Issue by pvgenuchten** [soilwise-he/Soilwise-userstory-epics#11]
As a researcher at [Earthwatch](https://earthwatch.org.uk) responsible for the [Farming with Nature](https://earthwatch.org.uk/program/farming-with-nature/) program. 

We want to augment our field data with external, verified information
We also look for ways to make better use of existing data on soils, management practices and model derived measurements.

This integration can significantly enhance the accuracy and validity of research outcomes within the Farming with Nature program. We can save resources but also accelerate research timelines and we gain a deeper insights into various environmental factors.

## Extended user story description

The Earthwatch Europe foundation is an NGO and independent research institute that is committed to a sustainable society. Monitoring the environment and pollution lies at the core of our work. Our team in the Netherlands focuses on the transition to sustainable agriculture, improving water quality and greening measures and climate adaptation.

Soil Health is key in agriculture, but also for cities and freshwater management. Existing tools are expensive, narrow or require advanced skills. We have developed a broad, entry-level tool that helps users observe and reflect on the condition and health of the soil. We have developed protocols and tested them for two years. The Soil Health Toolkit is currently applied in multiple projects across 9 countries. 

As a Research Lead at Earthwatch Europe, I oversee the Soil Health Toolkit within the Farming with Nature program. Our toolkit, launched in February '23, captures data on soil parameters and displays results on the Farming with Nature platform. We aim to optimise data quality and explore digital technologies like satellites and drones. We actively participate in the European benchmarking project, contributing to a transparent monitoring framework. The toolkit's goal is to provide an efficient feedback loop, enabling immediate discussions based on real-time data.

The Soil Health Toolkit uses five core parameters: soil texture, infiltration rate, soil colour, earthworm density and vegetation cover. These are the basic components that we capture in the application. The results are also visible in the dashboard. Our data viewer on the Farming with Nature platform allows us to learn and share results with our users, including instant feedback. An annual report card is under construction. The platform is versatile, we can easily add additional parameters.

With the toolkit we notice a trade-off between the amount of data and the quality of the data. You can generate a lot of data, but good quality data is much more useful. It is helpful that we have immediate access to the data. The feedback loop is short, we can immediately discuss results. Data quality is an important priority for us. Thanks to the amount of data, you could perhaps overcome this with statistical methods. 

We also invest in validation. We are involved in a [European BENCHMARKS project](https://soilhealthbenchmarks.eu/), which allows us to also perform proper soil tests and analysis in the lab. We saw nice similarities in organic matter and color. We try to validate as many parameters as possible in this way.

The BENCHMARKS project proposes the co-development within 24 European case studies of a multi-scale and multi-user focused monitoring framework that is transparent, harmonised and cost-effective. As EarthWatch we want to help build this European network to advance soil health research, monitor soil health and advocate for sustainable land use. Underpinned by the best scientific knowledge and technologies this framework provides a clear soil health index for benchmarking, using indicators that are pertinent to the objective of assessment, applicable to the land use and logistically feasible. Soil Health Benchmarks aims to validate and further develop indicators for soil health and functions.

## Acceptance criteria 
 
- [ ]	The system should seamlessly integrate open soil data with the Soil Health Toolkit, ensuring compatibility and accessibility.
- [ ]	Evaluate the potential of open soil data to enhance the toolkit's parameter coverage, going beyond the existing five core parameters.
- [ ]	Establish mechanisms to validate the quality of open soil data, ensuring it aligns with the high-quality standards set by the Soil Health Toolkit.
- [ ]	The integration of open soil data should contribute to an optimized feedback loop, enabling immediate discussions based on real-time data, enhancing the efficiency of decision-making.
- [ ]	Ensure that results from open soil data are visible in the dashboard and data viewer on the Farming with Nature platform, facilritating learning and instant feedback for users.
- [ ]	Explore the use of statistical methods to enhance the quality of open soil data.
- [ ]	Leverage open soil data for validation steps, especially in the context of the European benchmarking project, to perform proper soil tests and lab analysis.
- [ ]	The use of open soil data should contribute to the monitoring framework proposed by the benchmarking project and align with transparency, harmonization, and cost-effectiveness goals.

**Comment by roblokers** [comment]
Is there a way that at least use case leads can edit the issue (other than adding comments)

**Comment by pvgenuchten** [comment]
> Is there a way that at least use case leads can edit the issue (other than adding comments)

Yes, but you need to be a member of the Repository, request membership to radu, dajana or paul

**Comment by chlebas** [comment]
I don't see in this user story what is expected from SWR?

**Comment by KathiSchleidt** [comment]
Extracted Requirements:
- 01.1 - Data Discovery - Query Catalogue: topic query integrating relations from vocabularies and knowledge (e.g. including narrower terms and synonyms)
- 01.1 - Data Discovery - Query Catalogue: Find primary/derived data, real-time data, management practices
- 04 - Download data: primary/derived data, real-time data. data from satellites and drones
- 06 - Data quality checks: optimise data quality through short feedback loops, create checking mechanisms, Leverage open soil data for validation steps
- 07 - Harvesting: real-time verified data
- 08 - Vocabulary: Soil Vocabs
- 10 - Data harmonization: seamlessly integrate open soil data
- _11 - Processing: statistical methods to enhance the quality of open soil data (out-of-scope)_

**Comment by blitzfrayret** [comment]
> I don't see in this user story what is expected from SWR?

Me neither I do not understand the user need.
